UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -1741,22 +1833,44 @@ private void DeleteItem(UndertaleObject obj)
                     Data.GMLEditedBefore?.Remove(codeName);
                 }
 
-                CloseTab(obj);
+                while (CloseTab(obj)) ;
```

Comment:
CloseTab returns `void`, how does this while loop function?

Reply:
Check the line 3519 - `CloseTab(obj)` overload returns boolean.

Reply:
Missed that. That is *definitely* confusing though, since all other close tab functions only return void